src/components/admin/Settings/PrefixAlerts/Dialog/InitializationError.tsx

```diff
@@ -0,0 +1,70 @@
+import { AlertTitle, Box, Stack, Typography } from '@mui/material';
+import SafeLoadingButton from 'components/SafeLoadingButton';
+import AlertBox from 'components/shared/AlertBox';
+import { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { initializeNotificationSubscriptions } from '../useAlertSubscriptionDialog';
+import useAlertSubscriptionsStore from '../useAlertSubscriptionsStore';
+
+export default function InitializationError() {
+    const serverError = useAlertSubscriptionsStore(
+        (state) => state.serverError
+    );
+
+    const prefix = useAlertSubscriptionsStore((state) => state.prefix);
+
+    const initializeSubscriptionState = useAlertSubscriptionsStore(
+        (state) => state.initializeState
+    );
+
+    const [loading, setLoading] = useState(false);
+
+    const retryInitialization = async (
+        event: React.MouseEvent<HTMLElement>
+    ) => {
+        event.preventDefault();
```

Comment:
Unless these buttons are in tables (or any parent that is also listening for an event) this is not needed. These can be removed.

Reply:
Understood. This code closely mirrors the other instances of this custom error handling; I wanted to get a sense of the functional differences across them. Can remove.